### PR TITLE
flite: add page

### PR DESCRIPTION
**Pull request by Sadeeed**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Sadeeed** [comment]
#8461

**Comment by kbdharun** [comment]
@Sadeeed this file must be placed in `common` as it is multi-platform.

**Comment by Sadeeed** [comment]
@kbdharun Moved to common thank you for the heads up

File: pages/common/flite.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# flite
+
+> A small simple speech synthesizer.
+> More information: <http://www.festvox.org/flite/doc/index.html>.
```
Comment (navarroaxel) [inline pages/common/flite.md:4]:
```suggestion
> More information: <http://www.festvox.org/flite/doc/>.
```

File: pages/common/flite.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# flite
+
+> A small simple speech synthesizer.
```
Comment (navarroaxel) [inline pages/common/flite.md:3]: We prefer an objective description for the command
```suggestion
> Speech synthesis engine.
```

File: pages/common/flite.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <http://www.festvox.org/flite/doc/index.html>.
+
+- Show flite version information:
```
Comment (navarroaxel) [inline pages/common/flite.md:6]: This is almost and standard example. This should be the last one
```suggestion
- Display version:
```

File: pages/common/flite.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`flite -t "{{string}}"`
+
+- Convert the contents of a file to speech:
+
+`flite -f {{path/to/file}}`
+
+- Specify which voice to use:
+
+`flite -voice {{file://path/to/filename.flitevox | url}}`
```
Comment (navarroaxel) [inline pages/common/flite.md:24]: When the hostname is omitted the URI has 3 slashes `/`
```suggestion
`flite -voice {{file://path/to/filename.flitevox|url}}`
```

File: pages/common/flite.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`flite -lv`
+
+- Convert a text string to speech:
+
+`flite -t "{{string}}"`
+
+- Convert the contents of a file to speech:
+
+`flite -f {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/flite.md:20]: Should be a text file, so
```suggestion
`flite -f {{path/to/file.txt}}`
```

File: pages/common/flite.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Specify which voice to use:
+
+`flite -voice {{file://path/to/filename.flitevox | url}}`
+
+- Store output into a wav file:
+
+`flite -voice {{file://path/to/filename.flitevox | url}} -f {{path/to/textfile}} -o {{output.wav}}`
```
Comment (navarroaxel) [inline pages/common/flite.md:28]:
```suggestion
`flite -voice {{file://path/to/filename.flitevox|url}} -f {{path/to/file.txt}} -o {{output.wav}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**